metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -357,7 +384,7 @@ def construct_card_from_api_data(self, card_data: dict) -> Optional[ChartSnapsho
             lastModified=AuditStamp(time=modified_ts, actor=modified_actor),
         )
 
-        chart_type = self._get_chart_type(card_id, card_details.get("display"))
+        chart_type = self._get_chart_type(card_id, card_details.get("display", ""))
```

Comment:
```suggestion
        chart_type = self._get_chart_type(card_id, card_details.get("display") or "")
```

Reply:
Could you please explain why setting default value with `or` is better than passing `default` into `.get()` method?

Reply:
assuming here that chart_type is handled as str and not Optional[str] throughout the code.

if metabase api returns `card_details` as `{"display": None} `
then chart_type will not be a str anymore and can lead to errors down the code path.
We've run into this in a couple of places for tableau graphql api before. Metabase may be different but prefer to be defensive in case api behaves unreliably.

Reply:
Ah yes, I completely forgot about getting `None` by existing key.
Thank you for for this note! I've updated the code like suggested.